source/_posts/2020-09-14-birthday-week.markdown

```diff
@@ -0,0 +1,28 @@
+---
+title: "Happy seventh birthday Home Assistant! 🎉"
+description: "We're turning 7 this week and are celebrating this with a week of cool releases. Keep an eye on the blog for the latest."
+date: 2020-09-14 00:00:00
+date_formatted: "September 14, 2020"
+comments: true
+author: Paulus Schoutsen
+categories: Announcements
+og_image: /images/blog/2020-09-14-birthday-week/social.png
+---
+
+![Celebration graphic](/images/blog/2020-09-14-birthday-week/social.png)
+
+This week Home Assistant turns seven years old, wild! It all started with me automating my Hue lights when the sun was setting and someone was home, and now we’re here. 😎
+
+It's been a crazy busy year. We've seen our contributions increase since the pandemic started, and it hasn't lowered. To give some numbers, in the last 4 weeks, we have merged more than 1400 contributions! The core of Home Assistant has seen contributions from over 2100 different people at this point. And that doesn't include the work that is being done on our frontend, iOS app, Android app or any of the documentation websites.
+
+All these people are working to make Home Assistant the best home automation software out there. For anyone to enjoy for free. With our focus on local control and privacy, we give people worldwide the ability to live their life. No fear that your smart home stops working when a manufacturer decides to pull the plug. No fear that your life is shared with third parties to improve the experience of a light bulb. Your home, your data, your life.
+
+In the last year, we've seen Home Assistant get more user friendly, more powerful and receive significant performance improvements. We're continuing to be able to give everyone a great experience, regardless of expertise level.
+
+We’re going to celebrate it this year with not just one day, but with a whole week of releases. 🎁 So check back each day this week to catch the latest!
```

Comment:
Just to avoid someone thinks we are doing a Home Assistant Core release every day.

```suggestion
We’re going to celebrate it this year with not just one day, but with a whole week of daily announcements and various releases. 🎁 So check back each day this week to catch the latest!
```